addons/clones/addon.json

```diff
@@ -34,9 +44,8 @@
     }
   ],
   "latestUpdate": {
-    "version": "1.28.0",
-    "newSettings": ["showicononly"],
-    "isMajor": false
+    "version": "1.30.0",
+    "temporaryNotice": "You can now choose whether the clone counter shows up only in the editor or also on the project page."
```

Comment:
I don't think the notice is really necessary sense the new tag appears net to the setting. It it is kept, maybe it could be shortened a little?

Reply:
I also forgot to add the "New!" label next to the new setting. I'll do that instead.